osc.go

```diff
@@ -31,7 +31,7 @@ func (t *Terminal) handleOSC(code string) {
 
 func (t *Terminal) setDirectory(uri string) {
 	u, err := storage.ParseURI(uri)
-	if err == nil {
+	if err != nil {
 		// working around a Fyne bug where file URI does not parse host
```

Comment:
Is there an upstream bug tracking this?

Reply:
I'm not sure if it is still an issue or what the details are - it was so long ago.
But it doesn't relate to this fix ironically - the issue here was that it was using the url when the parse failed instead of when it succeeded...